src/deck/guides/vaults.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Secret Management with decK
+content_type: how-to
+---
+
+decK supports secret references for encoded values in {{site.base_gateway}}.
+You can store your secrets a [vault backend](/gateway/latest/kong-enterprise/secrets-management/),
+then reference them in your declarative configuration files.
+
+This guide covers storing values as secrets. You can use secrets to store sensitive data, such as credentials.
+
+## {{site.base_gateway}} secrets or decK environment variables?
+
+With decK, instead of checking credentials into Git, you can choose one of the following options:
+
+Option | Description | Why use this method?
+-------|-------------|---------------------
+[decK environment variables](/deck/latest/guides/environment-variables) | Store values as environment variables and access them directly through decK. {{site.base_gateway}} is not involved. | • You can use this option for environment-specific values. <br><br> • This method can store any configuration values used by {{site.base_gateway}} entities. It can't secure values from `kong.conf`. <br><br> • Available for all {{site.base_gateway}} packages: open-source, Enterprise Free mode, and Enterprise licensed mode.
+Secrets in {{site.base_gateway}} | Store values as secrets in a vault, then reference the secrets with a vault reference. In this case, {{site.base_gateway}} manages the secrets with a `vaults` entity. | • This option is more secure. <br><br> • You can use secrets to store many sensitive values, including parameters in Kong's configuration (`kong.conf`). See [Secrets Management in {{site.base_gateway}}](/gateway/latest/kong-enterprise/secrets-management/#what-can-be-stored-as-a-secret) for a full list. <br><br> • Secrets management is only available for {{site.base_gateway}} Enterprise packages. It is not available for open-source {{site.base_gateway}}. <br>The environment variable vault can be used in Free mode without a license, while all other vault backends require a license.
```

Comment:
```suggestion
Secrets in {{site.base_gateway}} | Store values as secrets in a vault, then reference the secrets with a vault backend. In this case, the {{site.base_gateway}} data plane manages the secrets with a `vaults` entity. |  • You can use secrets to store sensitive values, see [Secrets Management in {{site.base_gateway}}](/gateway/latest/kong-enterprise/secrets-management/#what-can-be-stored-as-a-secret) for a full list. <br><br> • Secrets management is only available for {{site.base_gateway}} Enterprise packages. It is not available for open-source {{site.base_gateway}}. <br>The environment variable vault can be used in Free mode without a license, while all other vault backends require a license.
```

Reply:
I think the license stuff should move to the description box. I think the why could stand to list the supported backends, site security independently, like "Is a secure way to manage sensitive information".